Makefile

```diff
@@ -64,3 +64,21 @@ clean:
 	"$(GO)" clean -cache -testcache -r
 	@echo "removing generated files"
 	@rm -rf build
+
+.PHONY: go-lint
+go-lint: ## run go linters.
+	@echo '>>> Running go linters.'
+	@golangci-lint run -v --issues-exit-code 0 ## TODO: remove after fixing all lint issues
+
+.PHONY: go-format
+go-format: ## format go code.
+	@echo '>>> Formatting go code.'
+	@gofumpt -w .
```

Comment:
for this step, we can just use `gofmt -w .` - it's part of the Go stdlib, and should already be included with the go compiler install.